topology/sof-apl-nocodec.m4

```diff
@@ -1,11 +1,11 @@
 #
-# Topology for generic Apollolake board with no codec.
+# Topology for generic Apollolake board with TDF8532
 #
```

Comment:
Error in the comment here? should be no codec?

Reply:
it was typo, thanks for pointing it out. let me update it.